src/pattern-library/components/patterns/prototype/TabbedDialogPage.tsx

```diff
@@ -141,7 +141,9 @@ function TabbedDialog() {
                 Three
               </Tab>
             </TabListHeader>
-            <Card>
+            <Card
+              classes={selectedTab === 'one' ? 'rounded-tl-none' : undefined}
```

Comment:
Unrelated to the changes here, this component is called "TabbedDialogPage" but that doesn't match the Import/Export label in the sidebar. We also might want to extract this pattern as a component in future to make it easier to instantiate?

Reply:
> We also might want to extract this pattern as a component in future to make it easier to instantiate?

Agree.

It would also make it easier to transparently handle the rounded corners when the first tab is selected, so that you don't need to remember every time this pattern is used.